### DISCUSSION TITLE
GitHub mobile app's Explore Page not showing anything

### DISCUSSION THREAD

**Discussion by blunweon** [community/community#146744] [Mobile] [answered]
<details>
Recently I found that applying filters (or not) does not work properly. I am unable to see repos or released that I have starred while previously I was able to. Posted screenshots below to show my experience:
![image](https://github.com/user-attachments/assets/c90e8ca3-9124-4715-821a-7a27a41a0c55)
![image](https://github.com/user-attachments/assets/7be41d0f-ef4a-46f1-b266-cb1c20a4ad8a)
As you can see, I am unable to see any releases or discussions at all.
<summary>ℹ️ Unable to see releases in GitHub iOS app</summary>
GitHub for iOS, Version 1.188.0 (148218644)
Device: iPhone 15 Pro (iOS 18.1.1)
</details>

**Comment by taki-on** [comment, marked as answer]
@blunweon Thank you for reporting the issue. I will file a bug and discuss within the team!

**Reply by blunweon** [reply]
Additional info: I can "re-set" filters and turn it off on the main website and see it on the phone sometimes for awhile. Then when I exit the app and come back, it can go missing again. Thank you for the prompt response however!

**Reply by taki-on** [reply]
Thank you for sharing the details! I will document it in our issue, too. It is very helpful!

**Reply by blunweon** [reply]
Hi, seems that the bug is still around. Please do have a look at it if possible, thanks!

**Reply by blunweon** [reply]
Bump, still no fixes or updates? @taki-on

**Reply by blunweon** [reply]
@taki-on Any updates on this matter?
